### ISSUE TITLE
basename doesn't work

### ISSUE THREAD

**Issue by jovesus** [FelixKrueger/TrimGalore#50]
Hi, it is a great tool. Thanks.

I have an issue for the latest `--basename` argument. Below is the command I run, but the resulting file have the original names, not my preferred ones.

```
trim_galore --paired --trim1 -o res --basename HC_1 CK_S36_R1.fastq.gz CK_S36_R1.fastq.gz
```

The trimming is done perfectly but the names are not changed.
Please tell me where I made mistakes.

**Comment by FelixKrueger** [comment]
It appears that the command works just fine if you don't specify an output directory. I will try to have it fixed soon. For the time being, you could just run: 

`trim_galore --paired --basename HC_1 CK_S36_R1.fastq.gz CK_S36_R1.fastq.gz`

which should work. By the way, unless you are intending to use Bowtie (1) you should lose the option `--trim1`.

**Comment by FelixKrueger** [comment]
This should now be fixed. Please clone the latest dev version, and let me know if it works in your hands. Best, Felix

Addressed here: ccfc4a9d9b4bed92671e3760d57463add16b8dee

**Comment by jovesus** [comment]
@FelixKrueger Thank you for your quick reply and fix. It works fine now.
